core/tests/test_utils.py

```diff
@@ -2818,7 +2840,15 @@ def get_response_without_checking_for_errors(
     def _parse_json_response(
         self, json_response: webtest.TestResponse, expect_errors: bool
     ) -> Any:
-        """Convert a JSON server response to an object (such as a dict)."""
+        """Convert a JSON server response to an object (such as a dict).
+
+        Args:
+            json_response: webtest.TestResponse. The test reponse.
+            expect_errors: bool. Whether errors are expected.
+
+        Returns:
+            Any. Python object.
```

Comment:
Please be more specific. The code seems to be returning some sort of json

Reply:
I have updated the docstring.